libs/community/langchain_community/vectorstores/milvus.py

```diff
@@ -133,6 +133,7 @@ def __init__(
         partition_names: Optional[list] = None,
         replica_number: int = 1,
         timeout: Optional[float] = None,
+        num_shards: Optional[int] = 1,
```

Comment:
does a default of 1 represent a change in behavior for users, who are currently not specifying num_shards?

Reply:
Default is 1, to avoid breaking changes for existing users,
Default value is 1 in pymilvus also, if we will not mention any number in num_shards while collection creation,
This to avoid None exception.

Reply:
We can remove the default value, but just to keep the same number with pymilvus , i just added the default 1